### PR TITLE
Add NCR401U

### PR DESCRIPTION
**Pull request by fvollmer**
Add NCR401U

Datasheet: https://assets.nexperia.com/documents/data-sheet/NCR401U.pdf

grafik

Travis complains about `LT3755xMSE` and `LT3755xUD`. I didn't change them, but KiCad modified them anyway (now all aliases are in one line).

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

File: Driver_LED.lib [hunk 1]
```diff
@@ -989,9 +989,7 @@ F0 "U" -550 750 50 H V C CNN
 F1 "LT3755xMSE" 350 750 50 H V C CNN
 F2 "Package_SO:MSOP-16-1EP_3x4mm_P0.5mm_EP1.65x2.85mm" 1000 -750 50 H I C CNN
 F3 "" -50 -1100 50 H I C CNN
-ALIAS LT3755xMSE-2
-ALIAS LT3756xMSE
-ALIAS LT3756xMSE-2
+ALIAS LT3755xMSE-2 LT3756xMSE LT3756xMSE-2
```
Comment (cpresser) [inline Driver_LED.lib]: There is something weird going on here. 
The previous change to this file was done in https://github.com/KiCad/kicad-symbols/pull/2257, which I also reviewed. I assume it was with a different KiCad version.
Your commit cleared that up.

**Comment by cpresser** [comment]
Hi, thanks for your contribution.
- [ ] The description should contain the package name (KLC S6.3)
- [ ] The footprint-filter should not contain the pin-count (KLC S5.3)

**Comment by fvollmer** [comment]
Thank you for the review!
I fixed the description and the footprint filter.

**Comment by cpresser** [comment]
Thanks!

**Pull request merged**